packages/adapters/src/process-instance-adapter.ts

```diff
@@ -31,101 +23,19 @@ class ProcessInstanceAdapter implements
 ILifeCycleAdapterMain,
 ILifeCycleAdapterRun,
 IComponent {
-    private monitorFifoPath?: string;
-    private controlFifoPath?: string;
-    private inputFifoPath?: string;
-    private outputFifoPath?: string;
-    private loggerFifoPath?: string;
-
-    private runnerStdin: PassThrough;
-    private runnerStdout: PassThrough;
-    private runnerStderr: PassThrough;
-    private monitorStream: DelayedStream;
-    private controlStream: DelayedStream;
-    private loggerStream: DelayedStream;
-    private inputStream: DelayedStream;
-    private outputStream: DelayedStream;
-
     logger: Logger;
 
     private runnerProcess?: ChildProcess;
-    private fifosDir?: string;
 
     constructor() {
-        this.runnerStdin = new PassThrough();
-        this.runnerStdout = new PassThrough();
-        this.runnerStderr = new PassThrough();
-        this.monitorStream = new DelayedStream();
-        this.controlStream = new DelayedStream();
-        this.loggerStream = new DelayedStream();
-        this.inputStream = new DelayedStream();
-        this.outputStream = new DelayedStream();
-
         this.logger = getLogger(this);
     }
 
     async init(): Promise<void> {
         // noop
```

Comment:
I see that in the Instance Adapters we always have the init function empty. I wonder whether we'll some initiation to do in the Kubernetes Instance Adapter. If we come to it and see we don't have anything to initiate, maybe we could remove it.

Reply:
I'll keep that in mind